<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="ArrayTypeMismatchException.xml" source-language="en-US" target-language="ru-RU">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5973b2b40d5674416e247fce16ebdf227be62660b.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">973b2b40d5674416e247fce16ebdf227be62660b</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-1.0,netstandard-1.1,netstandard-1.2,netstandard-1.3,netstandard-1.4,netstandard-1.5,netstandard-1.6,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.ArrayTypeMismatchException">
          <source>The exception that is thrown when an attempt is made to store an element of the wrong type within an array.</source>
          <target state="translated">Исключение, которое выдается при попытке сохранить в массиве элемент неподходящего типа.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source><ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> is thrown when the system cannot convert the element to the type declared for the array.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> возникает, когда система не может преобразовать элемент в тип, объявленный для массива.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>For example, an element of type <ph id="ph1">&lt;xref:System.String&gt;</ph> cannot be stored in an <ph id="ph2">&lt;xref:System.Int32&gt;</ph> array because conversion between these types is not supported.</source>
          <target state="translated">Например, элемент типа <ph id="ph1">&lt;xref:System.String&gt;</ph> не могут храниться в <ph id="ph2">&lt;xref:System.Int32&gt;</ph> массива, так как преобразование между этими типами не поддерживается.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>It is generally unnecessary for applications to throw this exception.</source>
          <target state="translated">Обычно необязателен для приложений, чтобы это исключение.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>The following Microsoft intermediate language (MSIL) instructions throw <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> :</source>
          <target state="translated">Корпорации Майкрософт, вызывать инструкции промежуточного языка (MSIL) <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> :</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>ldelem.<ph id="ph1">\&lt;</ph>type&gt;</source>
          <target state="translated">ldelem.<ph id="ph1">\&lt;</ph>type&gt;</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>ldelema</source>
          <target state="translated">ldelema</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>stelem.<ph id="ph1">\&lt;</ph>type&gt;</source>
          <target state="translated">stelem.<ph id="ph1">\&lt;</ph>type&gt;</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source><ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> uses the HRESULT COR_E_ARRAYTYPEMISMATCH, which has the value 0x80131503.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> использует COR_E_ARRAYTYPEMISMATCH HRESULT, который имеет значение 0x80131503.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>For a list of initial property values for an instance of <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>, see the <ph id="ph2">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> constructors.</source>
          <target state="translated">Список начальных значений свойств для экземпляра <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>, в разделе <ph id="ph2">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> конструкторы.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>The following code example demonstrates scenarios where <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> is thrown.</source>
          <target state="translated">В следующем примере кода показаны сценария, где <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> возникает исключение.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="T:System.ArrayTypeMismatchException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> class.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> class.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> property of the new instance to a system-supplied message that describes the error, such as "Source array type cannot be assigned to destination array type."</source>
          <target state="translated">Этот конструктор инициализирует <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> свойств нового экземпляра системным сообщением, содержащим описание ошибки, такие как «нельзя назначить тип исходного массива к типу массива назначения.»</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>This message takes into account the current system culture.</source>
          <target state="translated">Это сообщение учитывает текущую культуру системы.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>.</source>
          <target state="translated">В следующей таблице представлены исходные значения свойств экземпляра класса <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>Property</source>
          <target state="translated">Свойство.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>Value</source>
          <target state="translated">Значение</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>A null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic).</source>
          <target state="translated">Пустая ссылка (<ph id="ph1">`Nothing`</ph> в Visual Basic).</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>The localized error message string.</source>
          <target state="translated">Строка локализованного сообщения об ошибке.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>The following example demonstrates the ArrayTypeMismatchException() constructor of the <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> class.</source>
          <target state="translated">В следующем примере демонстрируется конструктор ArrayTypeMismatchException() <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> класса.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>It contains a function which takes two arrays as arguments and checks whether the two arrays are of the same type.</source>
          <target state="translated">Он содержит функцию, которая принимает в качестве аргументов два массива и проверяет, являются ли два массива того же типа.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>If the arrays are of different types, a new <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> is thrown and then caught in the calling method.</source>
          <target state="translated">Если массивы разных типов, новый <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> и затем перехватывается исключение в вызывающем методе.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>A <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph> that describes the error.</source>
          <target state="translated">Строка <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph>, описывающая ошибку.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> class with a specified error message.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> с указанным сообщением об ошибке.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>The content of the <ph id="ph1">`message`</ph> parameter is intended to be understood by humans.</source>
          <target state="translated">Содержимое параметра <ph id="ph1">`message`</ph> должно быть понятно пользователям.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">Код, вызывающий этот конструктор, должен обеспечить локализацию данной строки в соответствии с текущим языком и региональными параметрами системы.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>.</source>
          <target state="translated">В следующей таблице представлены исходные значения свойств экземпляра класса <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>Property</source>
          <target state="translated">Свойство.</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>Value</source>
          <target state="translated">Значение</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>A null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic).</source>
          <target state="translated">Пустая ссылка (<ph id="ph1">`Nothing`</ph> в Visual Basic).</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>The error message string.</source>
          <target state="translated">Строка сообщения об ошибке.</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>The following example demonstrates the ArrayTypeMismatchException(String) constructor of the <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> class.</source>
          <target state="translated">В следующем примере демонстрируется конструктор ArrayTypeMismatchException(String) <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> класса.</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>It contains a function which takes two arrays as arguments and checks whether the two arrays are of the same type.</source>
          <target state="translated">Он содержит функцию, которая принимает в качестве аргументов два массива и проверяет, являются ли два массива того же типа.</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>If the arrays are of different types, a new <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> is thrown and then caught in the calling method.</source>
          <target state="translated">Если массивы разных типов, новый <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> и затем перехватывается исключение в вызывающем методе.</target>       </trans-unit>
        <trans-unit id="136" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The object that holds the serialized object data.</source>
          <target state="translated">Объект, содержащий сериализованные данные объекта.</target>       </trans-unit>
        <trans-unit id="137" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The contextual information about the source or destination.</source>
          <target state="translated">Контекстные сведения об источнике или назначении.</target>       </trans-unit>
        <trans-unit id="138" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> class with serialized data.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> с сериализованными данными.</target>       </trans-unit>
        <trans-unit id="139" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</source>
          <target state="translated">Данный конструктор вызывается в процессе десериализации для воссоздания объекта исключения, переданного в потоке.</target>       </trans-unit>
        <trans-unit id="140" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>For more information, see <bpt id="p1">[</bpt>XML and SOAP Serialization<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>.</source>
          <target state="translated">Дополнительные сведения см. в разделе <bpt id="p1">[</bpt>сериализации XML и SOAP<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>.</target>       </trans-unit>
        <trans-unit id="141" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Сообщение об ошибке, указывающее причину создания исключения.</target>       </trans-unit>
        <trans-unit id="142" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">Исключение, которое является причиной текущего исключения.</target>       </trans-unit>
        <trans-unit id="143" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>If the <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> parameter is not a null reference, the current exception is raised in a <ph id="ph1">&lt;see langword="catch" /&gt;</ph> block that handles the inner exception.</source>
          <target state="translated">Если параметр <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> не является пустой ссылкой, текущее исключение возникает в блоке <ph id="ph1">&lt;see langword="catch" /&gt;</ph>, который обрабатывает внутреннее исключение.</target>       </trans-unit>
        <trans-unit id="144" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> указанным сообщением об ошибке и ссылкой на внутреннее исключение, вызвавшее данное исключение.</target>       </trans-unit>
        <trans-unit id="145" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property.</source>
          <target state="translated">Исключение, созданное как прямой результат предыдущего исключения, должно содержать в свойстве <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> ссылку на предыдущее исключение.</target>       </trans-unit>
        <trans-unit id="146" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed into the constructor, or a null reference (<ph id="ph2">`Nothing`</ph> in Visual Basic) if the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated">Свойство <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> возвращает то же значение, которое передается конструктору, или пустую ссылку (<ph id="ph2">`Nothing`</ph> в Visual Basic), если свойство <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> не предоставляет конструктору значение внутреннего исключения.</target>       </trans-unit>
        <trans-unit id="147" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>.</source>
          <target state="translated">В следующей таблице представлены исходные значения свойств экземпляра класса <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="148" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>Property</source>
          <target state="translated">Свойство.</target>       </trans-unit>
        <trans-unit id="149" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>Value</source>
          <target state="translated">Значение</target>       </trans-unit>
        <trans-unit id="150" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>The inner exception reference.</source>
          <target state="translated">Ссылка на внутреннее исключение.</target>       </trans-unit>
        <trans-unit id="151" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>The error message string.</source>
          <target state="translated">Строка сообщения об ошибке.</target>       </trans-unit>
        <trans-unit id="152" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> class.</source>
          <target state="translated">В следующем примере кода показано <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException.%23ctor%2A&gt;</ph> конструктор <ph id="ph2">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> класса.</target>       </trans-unit>
        <trans-unit id="153" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>It contains a function that takes two arrays as arguments and checks whether the two arrays are of the same type.</source>
          <target state="translated">Он содержит функцию, которая принимает в качестве аргументов два массива и проверяет, являются ли два массива того же типа.</target>       </trans-unit>
        <trans-unit id="154" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>If the arrays are of different types, a new <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> is thrown and then caught in the calling method.</source>
          <target state="translated">Если массивы разных типов, новый <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> и затем перехватывается исключение в вызывающем методе.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>